### PR TITLE
fix: changed many rules in browser-to-appname map to use case-insensitive regexes

### PR DESCRIPTION
**Pull request by ErikBjare**
Replaces https://github.com/ActivityWatch/aw-webui/pull/283

| :rocket: This description was created by [Ellipsis](https://www.ellipsis.dev) for commit f53f8b8b2afdbf3447b4156bb1d25686ddf0f77a  | 
|--------|

### Summary:
Enhanced the `browser_appnames` mapping in `/src/queries.ts` with improved regex patterns for flexible, case-insensitive browser identification, including updates for `LibreWolf`, `Brave`, and other browsers.

**Key points**:
- Updated `browser_appnames` in `/src/queries.ts` to use regex for flexible, case-insensitive matching.
- Regex patterns handle variations in browser names across platforms and versions.
- Enhanced matching for `LibreWolf` with new variant `librewolf-default`.
- Improved regex patterns for `Brave` and other browsers.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

File: src/queries.ts [hunk 1]
```diff
@@ -213,75 +213,48 @@ export function appQuery(
...
-    'Chromium-browser-chromium',
-    'chromium.exe',
-
-    // Pre-releases
-    'Google-chrome-beta',
-    'Google-chrome-unstable',
+    /chromium\w?(browser)?(-chromium)?(.exe)?/i,
 
     // Brave (should this be merged with the brave entry?)
     'Brave-browser',
```
Comment (BelKed) [inline src/queries.ts:227]: Just curious — Is there some reason for not moving these lines to the Brave Browser category?

Comment (ErikBjare) [reply src/queries.ts:227]: I don't remember...

Found https://discord.com/channels/755040852727955476/755334543891759194/1237102962241704038

Looks like we should merge: https://github.com/ActivityWatch/aw-webui/pull/283

But something stalled it.

**Comment by ErikBjare** [comment]
Looks like I can't pass regexes? Makes sense if the mapping goes into the query...

**Comment by ErikBjare** [comment]
This will never work.

**Pull request closed without merging**